### PR TITLE
9 unit of measure controller

### LINKED ISSUE #9: Unit of measure controller
Create a controller to manage the unit of measures

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**
Closes #9

### CODE REVIEW

**Comment by josephbridgwaterrowe** [comment]
@michaelachrisco please review.

File: app/views/fagus/categories/show.html.erb [hunk 1]
```diff
@@ -1,5 +1,3 @@
-<%= flash_messages %>
```
Comment (michaelachrisco) [inline app/views/fagus/categories/show.html.erb:1]: Why did you remove the flash message here?

Comment (josephbridgwaterrowe) [inline app/views/fagus/categories/show.html.erb:1]: I moved it to the layout in a previous branch :smile:

Comment (michaelachrisco) [inline app/views/fagus/categories/show.html.erb:1]: :+1:

**Comment by michaelachrisco** [comment]
Fair enough. Rebase and I will merge.

**Comment by michaelachrisco** [comment]
Rebase?

**Comment by josephbridgwaterrowe** [comment]
This is kind of dead now, but I will rebase to tie up loose ends.

**Comment by josephbridgwaterrowe** [comment]
Thanks.
